include/mruby.h

```diff
@@ -263,28 +263,22 @@ MRB_API void mrb_define_method(mrb_state *mrb, struct RClass *cla, const char *n
  * Defines a class method.
  *
  * Example:
- *   # Ruby style
- *   class Foo
  *
- *     def Foo.bar
+ *     # Ruby style
+ *     class Foo
+ *      def Foo.bar
```

Comment:
It's one space indentation? I'm using phone. I think another space is needed for indentation.